### ISSUE TITLE
Assemble Redesign

### ISSUE THREAD

**Issue by grayghostvisuals** [node-toolkit/toolkit#1]
Desired Message: How truly powerful these tools are, and how they allow “humans to scale themselves”, and demonstrate that through concrete stats and achievements.

Artwork https://github.com/node-toolkit/artwork

What adjectives are used when one thinks about Assemble?
- powerful
- scalable

Pages
- Home

Possible Font Choices:
- https://fonts.google.com/specimen/Nunito+Sans
- https://fonts.google.com/specimen/Palanquin

**Comment by jonschlinkert** [comment]
k, before I add this to a doc or push up the rest of the docs, what are your thoughts on [this overview/sitemap](https://gist.github.com/jonschlinkert/5c349ce5590fbe1ec65b3de911cc5914)? Your feedback might impact how we finish organizing the content before I push it up. so it's not all over the place

**Comment by grayghostvisuals** [comment]
Yep. I think that is a great start. Everything can be altered or adjusted as u go, but it's states the direction and purpose.

**Comment by jonschlinkert** [comment]
Great, I'm going to create a repo on toolkit called `docs/`. This will be a temporary repo that @doowb and I will use to collate information. I'll explain this in more detail on the readme of that repo.

Also, @doowb and I will focus on actual markdown documentation and organization, including collating, information structure and so on. We can also do the build and write any plugins or additional tools that are needed for search, navigation, pagination, etc. We'll try to generalize as much of that as possible so that it can be used in external plugins etc.

All/any of that sound good to you?

edit: also fwiw, once I push this stuff up, you can ignore anything that isn't related to assemble if you want. we created these projects to be completely standalone. @doowb and I just need to un-tangle some docs across all the related projects for our own sanity, to help us finish up the assemble docs. but please don't let that confuse what you wanted to accomplish or make this seem more complicated than it really is.  (also, @doowb is going to be working on search tools that will allow users to find information for any of the toolkit projects, on any of the toolkit projects, and across any of the toolkit projects. and this will help)

**Comment by jonschlinkert** [comment]
k, [here is the docs repo I mentioned](https://github.com/node-toolkit/docs). 

To be clear, this is for @doowb and me to use for collating information. You do not need to use that repo, but it's where we're going to be untangling everything.
